srcCs/RTIDDSImpl.cs

```diff
@@ -1769,7 +1758,7 @@ public IMessagingWriter CreateWriter(string topic_name)
             }
 
             // These QOS's are only set for the Throughput datawriter
-            if ((qos_profile == "ThroughputQos") || (qos_profile == "NoAckThroughputQos"))
+            if (qos_profile == "ThroughputQos")
             {
```

Comment:
"{" can be moved to the line before

Reply:
I left it there for consistency with the rest of the code.